python/GafferTest/ScriptNodeTest.py

```diff
@@ -1316,18 +1316,20 @@ def testFileNameInExecutionError( self ) :
 
 		for method in ( s.load, functools.partial( s.executeFile, fileName ) ) :
 
-			self.assertRaisesRegex(
-				RuntimeError,
-				"Line 2 of {} : NameError: name 'iDontExist' is not defined".format( fileName ),
-				method
-			)
+			with self.subTest( fileName = fileName, method = method ) :
+
+				self.assertRaisesRegex(
+					RuntimeError,
+					"Line 2 of {} : NameError: name 'iDontExist' is not defined".format( fileName.as_posix() ),
```

Comment:
Presumably this is needed because the internal error reporting in ScriptNode uses `filesystem::path::generic_string()`? I noticed that we're not consistent with that, with [other errors](https://github.com/GafferHQ/gaffer/blob/main/src/Gaffer/ScriptNode.cpp#L220) just using `string()`. Maybe there's a reasonable argument for using `string()`, since that's what we would have passed to the underlying OS calls? Not something we need to change on this PR, but I wonder if this is something we should standardise a little more in future...